lib/datadog/appsec/extensions.rb

```diff
@@ -88,6 +88,10 @@ def obfuscator_value_regex=(arg)
           @settings.merge(dsl)
         end
 
+        def default?(option)
+          !@settings.default?(option)
+        end
```

Comment:
Is the `!` here correct? This seems to be negating the inner default, which doesn't seem correct? (Also, should there be tests to catch this?)